### ISSUE TITLE
Plugins have not been migrated to CodeIgniter3 / php7

### ISSUE THREAD

**Issue by tenzap** [back2arie/Kalkun#201]
title says it all

I tried renaming the controllers files of each plugin by making the first letter capitalized, but that's not sufficient, now it returns errors about models, deprecated php constructors and so on...

BTW, changing the 1st letter of the constrollers to capitalized, requires to modify also `application/views/main/plugin/index.php` (this has been added to PR #200 in the meantime ) so that the plugin gets an anchor when in the plugin view.
```
-               if ($type=="installed" AND file_exists(APPPATH . "plugins/".$tmp['plugin_system_name']."/controllers/".$tmp['plugin_system_name'].".php"))
+               if ($type=="installed" AND file_exists(APPPATH . "plugins/".$tmp['plugin_system_name']."/controllers/".ucfirst($tmp['plugin_system_name']).".php"))

```

**Comment by tenzap** [comment]
Should be fixed by PR #200

**Comment by tenzap** [comment]
closing since #200 is merged
